src/com/lftechnology/java/training/alina/basic/guessnumber/Randomize.java

```diff
@@ -8,18 +8,18 @@
  * @author Alina Shakya <alinashakya@lftechnology.com>
  */
 public class Randomize {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1118)